Description: Remove include/agg_conv_gpc.h from build files since it has been
  removed from the source tarball.
Author: John Horigan <john@glyphic.com>

--- a/include/Makefile.am
+++ b/include/Makefile.am
@@ -27,7 +27,7 @@
 	agg_conv_contour.h           agg_rasterizer_scanline_aa.h    agg_trans_affine.h \
 	agg_conv_curve.h             agg_rasterizer_sl_clip.h        agg_trans_bilinear.h \
 	agg_conv_dash.h              agg_renderer_base.h             agg_trans_double_path.h \
-	agg_conv_gpc.h               agg_renderer_markers.h \
+	agg_renderer_markers.h \
 	agg_conv_marker.h            agg_renderer_mclip.h            agg_trans_perspective.h \
 	agg_conv_marker_adaptor.h    agg_renderer_outline_aa.h       agg_trans_single_path.h \
 	agg_conv_segmentator.h       agg_renderer_outline_image.h    agg_trans_viewport.h \
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -24,7 +24,6 @@
     ${antigrain_SOURCE_DIR}/include/agg_conv_contour.h
     ${antigrain_SOURCE_DIR}/include/agg_conv_curve.h
     ${antigrain_SOURCE_DIR}/include/agg_conv_dash.h
-    ${antigrain_SOURCE_DIR}/include/agg_conv_gpc.h
     ${antigrain_SOURCE_DIR}/include/agg_conv_marker.h
     ${antigrain_SOURCE_DIR}/include/agg_conv_marker_adaptor.h
     ${antigrain_SOURCE_DIR}/include/agg_conv_segmentator.h
